### PR TITLE
update import of UO 

### PR DESCRIPTION
**Pull request by stap-m**
based on #14 and https://github.com/OpenEnergyPlatform/ontology/issues/1690
- update list of imported units and prefixes (uo-w-hierarchy.txt)
- update import and annotation script, such that it can be run by windows (extract-uo-module.sh)
- run script and thereby create new version of uo-extracted.owl
- remove uo-full-download.owl

### CODE REVIEW

File: oeo-imports/uo/uo-extracted.owl
Comment (christian-rli) [inline oeo-imports/uo/uo-extracted.owl]: I tried to open this link in a browser and it failed. Changing several parts of the link also doesn't fix it. Ontobee is currently down, which complicates proposing a fix.

Comment (stap-m) [reply oeo-imports/uo/uo-extracted.owl]: UO uses "rdfs:comment" as annotation property for definitions. We want to change this to "IAO_0000115" (definition), to be conform to OEO. This is done for all imported classes.

Comment (l-emele) [reply oeo-imports/uo/uo-extracted.owl]:
```suggestion
        <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000115"/>
```
The problem is the `#` character in the URL. Without that, the URL resolves to: https://ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115

**Review by l-emele — APPROVED** [review]
As far as I can tell this PR looks fine, apart from the `#` character where there is a specific comment. 

I can confirm that the uo-extracted.owl opens properly in Protégé and I do not spot any other problems. So I approve this PR, assuming that you will fix the line with the IRI discussion for IAO_0000115 before merging.

**Comment by stap-m** [comment]
I tried it out, this annotation comes automatically from ROBOT, see line 22 in the shell script. 
It appears where the definition annotation is annotated again with the "derived from" annotation. 
Actually, I don't want to mess with ROBOT and I don't see any harm for the import, since protege does not complain. Or do you see any reason to change anyway? @l-emele @christian-rli 
EDIT: it occurs in all entities btw, not just this line

**Comment by l-emele** [comment]
> Actually, I don't want to mess with ROBOT and I don't see any harm for the import, since protege does not complain.

Protégé complains only if it is syntactically correct. Protégé does not check whether every IRI is correct. 

> Or do you see any reason to change anyway?

Have you tried what happens if the current state of uo-extracted.owl is imported in the OEO? Does Protégé show the `definition` annotation of classes imported from UO with this wrong IRI correctly as it does for all other classes?

File: oeo-imports/uo/extract-uo-module.sh
Comment (l-emele) [inline oeo-imports/uo/extract-uo-module.sh]: I reviewed your script again. I think this line causes the problem with the additional `#`. What happens if you delete it here?
```suggestion
(Get-Content -ReadCount 0 uo-temp-extracted.owl) -replace 'xmlns:owl', 'xmlns:obo="http://purl.obolibrary.org/obo/"
```

Comment (stap-m) [reply oeo-imports/uo/extract-uo-module.sh]: Ok, seems to work. The prefix can live without the # 👍

**Comment by stap-m** [comment]
ToDo: we change the definition in OEO for unit, therefore we should remove the imported definition in uo-extracted in the shell script

**Comment by areleu** [comment]
From OEO meeting 82: 
Remove the definition from unit from the units ontology? 

or 

Remove Unit from import and import only the subclasses?

**Pull request merged**
